examples/passporting/src/app/api/shared-credential/[userId]/route.ts

```diff
@@ -3,20 +3,20 @@ import { getGrants } from "@idos-network/core/kwil-actions";
 
 export async function GET(request: Request, { params }: { params: Promise<{ userId: string }> }) {
   const userId = (await params).userId;
-
   const consumer = await idOSConsumer();
-  // @todo: filter grants by user id
-  const grants = await getGrants(consumer.kwilClient, 1);
-  const credentials = grants.find((grant) => grant.ag_owner_user_id === userId);
 
-  if (!credentials) {
+  const grants = await getGrants(consumer.kwilClient, {
+    user_id: userId,
+  });
+  const dag = grants.find((grant) => grant.ag_owner_user_id === userId);
```

Comment:
Please don't call this a dag, it feels very confusing. This is an AG, not a message.